clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -1925,6 +1926,56 @@ public void testCommittedAuthenticationFailure() {
         assertThrows(AuthenticationException.class, () -> consumer.committed(Collections.singleton(tp0)).get(tp0));
     }
 
+    @Test
+    public void testMeasureCommitSyncDuration() {
+        // use a consumer that will throw to ensure we return quickly
```

Comment:
I think we also need to test the case without failure. Otherwise, we assume in the test that the measurement is in the `finally` clause which we should not assume but rather ensure with unit tests. Same applies to the test below.